vector/src/main/java/im/vector/app/features/voicebroadcast/VoiceBroadcastConstants.kt

```diff
@@ -16,5 +16,11 @@
 
 package im.vector.app.features.voicebroadcast
 
-/** Voice Broadcast State Event. */
-const val STATE_ROOM_VOICE_BROADCAST_INFO = "io.element.voice_broadcast_info"
+object VoiceBroadcastConstants {
+
+    /** Voice Broadcast State Event. */
+    const val STATE_ROOM_VOICE_BROADCAST_INFO = "io.element.voice_broadcast_info"
+
+    /** Default voice broadcast chunk duration, in seconds. */
+    const val DEFAULT_CHUNK_LENGTH = 30
```

Comment:
Could be nice for clarity where it's used to have the unit in the name

```suggestion
    const val DEFAULT_CHUNK_LENGTH_IN_SECONDS = 30
```

Reply:
https://github.com/vector-im/element-android/pull/7363/commits/b9335c60657a114104a62e7a8a59e9ccb0b28c6e